R/seqSimVis.R

```diff
@@ -0,0 +1,98 @@
+
+
+#' \code{seqSimVis()} Calculates sequence similarity and plots dot plots based on the
+#' three highest similarity values to visualize
+#'
+#'
+#' @param gene_of_int (char) gene of interest
+#' @param system (char) system of interest
+
+#' @examples
+#' seqSimVis("BRCA1", "PHALY")
```

Comment:
Please add a comment explaining what your example code does.

Reply:
... so: where is the comment?